Rhino8DebugLauncher/Rhino8DebugLauncher.csproj

```diff
@@ -0,0 +1,14 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>WinExe</OutputType>
+    <TargetFramework>net7.0</TargetFramework>
```

Comment:
I think i made a mistake above: .net7.0 refers to the CORE version, while 4.8 refers to the .Net Framework version. 

Since we still want to have support for Rhino 7, we need to keep the 4.8, for the plugin, and the Core .net 7.0 is used to test on Rhino 8. I should have replaced 4.5.2 by 4.8, and added .net 7.0 to the list of dependencies.

Reply:
Ok, let's leave this for now. We should lift the debug launcher to .NET 7.0 as well if possible.